app/presenters/arclight/show_presenter.rb

```diff
@@ -5,6 +5,7 @@ module Arclight
   class ShowPresenter < Blacklight::ShowPresenter
     def heading
       title = super
+      title = nil if title.blank? || title == @document.id.to_s
```

Comment:
Can we return the id if there is no date or title? per https://github.com/sul-dlss/arclight/pull/280#issuecomment-299603090